tests/snuba/api/endpoints/test_group_events.py

```diff
@@ -61,6 +61,66 @@ def test_simple(self):
             [str(event_1.event_id), str(event_2.event_id)]
         )
 
+    def test_full_false(self):
```

Comment:
Think we should add a test for `full` not being passed at all?

Reply:
Added an assertion to the test_simple test case to ensure that it returns the simplified response when no params are passed